pages/linux/pyrit.md

```diff
@@ -0,0 +1,36 @@
+# pyrit
+
+> WPA/WPA2 cracking tool using computational power.
+> More information: <https://github.com/JPaulMora/Pyrit>.
+
+- Display system cracking speed:
+
+`pyrit benchmark`
+
+- List available cores:
+
+`pyrit list_cores`
+
+- Set ESSID:
+
+`pyrit -e {{ESSID}} create_essid`
```

Comment:
```suggestion
- Set [e]SSID:

`pyrit -e "{{ESSID}}" create_essid`
```

Optional fix: mnemonic can be added.
Optional fix: quoting can be added if the placeholder can contain special globbing or other characters to POSIX-compatible shell.